packages/affiliates/gas-rebate/VoterGasRebateV2.ts

```diff
@@ -93,6 +104,35 @@ export async function run(): Promise<void> {
     searchConfig
   );
 
+  // Filter out events with less than the minimum UMA tokens staked
+  const revealEventsMinBalance = revealEvents.filter((event) => event.args.numTokens.gte(minTokens));
+
+  // Filter out duplicate commit events as we only refund the latest commit event per voter per round
+  const uniqueCommitEvents = new Map<string, VoteCommittedEvent>();
```

Comment:
this is interesting. ive had to sometimes commit more than once because theres so many votes sometimes. is this a technical or design decision